### ISSUE TITLE
monetary policy

### ISSUE THREAD

**Issue by owocki** [gitcoinco/ERC-1789#28]
how do we present the "monetary policy of the core devs" ?  this is something that i'm putting in 'airquotes' for now, but i'm thinking it's a convo we'll neeed to have as this progresses

**Comment by lrettig** [comment]
Core devs are not economists, I don't think they're the right stakeholder group to determine this, at least not unilaterally. Yes, it's been that way historically but we can do better. It should be core devs in consultation with economists, or better yet, a [steering committee](https://github.com/gitcoinco/ERC-1789/issues/22) that includes people with the requisite skills and experience - with community input of course.

**Comment by owocki** [comment]
@lrettig i see you linked to the 'technical steering committee' ticket.. should we form a ticket for 'economic steering committee'?  and if so, what archectype of a person (and examples) of them should there be?

**Comment by lrettig** [comment]
Yes, an economic steering committee would be amazing. I don't think it has to be as large or as trusted as the technical steering committee since in my mind Ethereum is and probably will be a technocracy and the core devs will probably be the final arbiters for some time - so the output of this committee would be a set of reports and/or proposals that the core devs would review. I think 3-5 folks with experience in macroeconomics, both academic as well as practical, would be amazing.

Do you think we could convince a former central banker to join? 🤓

**Comment by jpitts** [comment]
A council or steering committee can perform research and publish findings on the anticipated effects of EIPs & groups of EIPs, and also send signals. Such a group can also point out misconceptions or identify emerging phenomena which need to be addressed by the community. 

Key to maintaining the integrity of these signals or findings is also publishing dissenting opinions.

I think that having such a group would improve the quality of decisions in the protocol.
